lib/benchmarks.nix

```diff
@@ -196,17 +196,19 @@ in rec {
   */
   mkBenchLWAFTR = { snabb
                   , times
-                  , duration
-                  , mode
-                  , ipv4PCap
-                  , ipv6PCap
-                  , conf
+                  , duration ? "10"
+                  , mode ? "bare"
+                  , ipv4PCap ? "ipv4-0550.pcap"
+                  , ipv6PCap ? "ipv6-0550.pcap"
+                  , conf ? "icmp_on_fail.conf"
+                  , loadTestStep ? "0.2e9"
```

Comment:
I had left these parameters without defaults to avoid duplicating the ones in `jobsets/snabb-lwaftr.nix`, which calls this. Are the defaults useful here too?